docs/operations-guide/running-the-metabase-jar-file.md

```diff
@@ -2,47 +2,134 @@
+
+Assuming you downloaded to `/Users/person/Downloads`:
+
+```
+mkdir metabase
+```
+
+then
+
+```
+mv /Users/person/Downloads/metabase.jar ~/metabase
+```
+
+### 4. Change into your new Metabase directory and run the jar
+
+Change into the directory you created in step 2:
+
+```
+cd ~/metabase
+```
 
 Now that you have Java working you can run the JAR from a terminal with:
 
-    java -jar metabase.jar
+```
+java -jar metabase.jar
+```
 
-It's that simple. This will start the Metabase application using all of the default settings. You should see some log entries starting to run in your terminal window showing you the application progress as it starts up. Once Metabase is fully started you'll see a confirmation such as:
+Metabase will start using the default settings. You should see some log entries starting to run in your terminal window showing you the application progress as it starts up. Once Metabase is fully started you'll see a confirmation such as:
 
-    ...
-    06-19 10:29:34 INFO metabase.task :: Initializing task CheckForNewVersions
-    06-19 10:29:34 INFO metabase.task :: Initializing task SendAnonymousUsageStats
-    06-19 10:29:34 INFO metabase.task :: Initializing task SendAbandomentEmails
-    06-19 10:29:34 INFO metabase.task :: Initializing task SendPulses
-    06-19 10:29:34 INFO metabase.task :: Initializing task SendFollowUpEmails
-    06-19 10:29:34 INFO metabase.task :: Initializing task TaskHistoryCleanup
-    06-19 10:29:34 INFO metabase.core :: Metabase Initialization COMPLETE
+```
+...
+06-19 10:29:34 INFO metabase.task :: Initializing task CheckForNewVersions
+06-19 10:29:34 INFO metabase.task :: Initializing task SendAnonymousUsageStats
+06-19 10:29:34 INFO metabase.task :: Initializing task SendAbandomentEmails
+06-19 10:29:34 INFO metabase.task :: Initializing task SendPulses
+06-19 10:29:34 INFO metabase.task :: Initializing task SendFollowUpEmails
+06-19 10:29:34 INFO metabase.task :: Initializing task TaskHistoryCleanup
+06-19 10:29:34 INFO metabase.core :: Metabase Initialization COMPLETE
+```
 
 At this point you're ready to go! You can access your new Metabase server on port 3000, most likely at [http://localhost:3000](http://localhost:3000)
 
 You can use another port than 3000 by setting the `MB_JETTY_PORT` [environment variable](./environment-variables.md) before running the jar.
 
-Note that in the default configuration Metabase will use a local H2 database for storing all its own application data. This is meant for simple evaluations or personal use, so if you want to run Metabase in production we recommend you [migrate away from H2](./migrating-from-h2.md).
+Note that in the default configuration Metabase will use a local H2 database for storing all its own application data. This default is meant for simple evaluation or personal use. If you want to run Metabase in production we recommend you [migrate away from H2](./migrating-from-h2.md).
+
+## Production installation
```

Comment:
We definitely want people to use a service instead of manually running a JAR. The service makes sure stuff runs automatically, but also creates a separate user with lower privileges which is good for security.
https://www.metabase.com/docs/latest/operations-guide/running-metabase-on-debian.html